infrastructure/tests/baloon-pods.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: placeholder
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      pod: placeholder-pod
+  template:
+    metadata:
+      labels:
+        pod: placeholder-pod
+    spec:
+      priorityClassName: placeholder-priority
+      terminationGracePeriodSeconds: 0
+      containers:
+      - name: ubuntu
+        image: ubuntu
+        command: ["sleep"]
+        args: ["infinity"]
+        resources:
+            requests:
+              cpu: 200m
```

Comment:
Can the CPU requests be really small (10m?) or not specfied at all?

Reply:
Ah, I guess if not specified it'll default to 500m in Autopilot. How about 10m or 25m?

Reply:
Fixed to 10m